src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -295,9 +295,10 @@ void func_808F3690(EnIn* this, GlobalContext* globalCtx) {
 }
 
 void func_808F374C(EnIn* this, GlobalContext* globalCtx) {
-    AnimationHeader* animations[] = { &object_in_Anim_015E38, &object_in_Anim_016A60, &object_in_Anim_0177AC,
-                                      &object_in_Anim_016484, &object_in_Anim_0170DC, &object_in_Anim_018240,
-                                      &object_in_Anim_0187C8, &object_in_Anim_0198A8 };
+    AnimationHeader* sAnimations[] = {
+        &object_in_Anim_015E38, &object_in_Anim_016A60, &object_in_Anim_0177AC, &object_in_Anim_016484,
+        &object_in_Anim_0170DC, &object_in_Anim_018240, &object_in_Anim_0187C8, &object_in_Anim_0198A8,
+    };
```

Comment:
It was my understanding that in-function static/array data does not get the `s` prefix, but I'm not sure if that's a hard-fast rule

Reply:
Ah, I had not heard that and was under the impression that all static variables should have `s` as a prefix, though it does make sense for in function to not due to there scope being just the function. However, this actor still needs a full doc pass, so it can probably just be addressed then when we have decided (hopefully :) ).

Reply:
There was some discussion about this in OoT, which didn't reach a conclusion. I think we might as well stick to `s` on all file-local data.

Reply:
I agree with Elliptic. I think it makes sense to be consistent and use it for all TU scoped data.